<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cass80Main</class>
 <widget class="QMainWindow" name="Cass80Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cassette Manager for TRS80 and EG2000</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="gb_listing">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Listing</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QTextEdit" name="te_listing">
          <property name="font">
           <font>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="gb_text">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Log output</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="tb">
          <property name="font">
           <font>
            <family>Source Code Pro</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="font">
     <font>
      <family>FreeSans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load"/>
    <addaction name="action_Save"/>
    <addaction name="action_Information"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo_lmoffset"/>
    <addaction name="separator"/>
    <addaction name="action_Preferences"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Load">
   <property name="icon">
    <iconset resource="cass80.qrc">
     <normaloff>:/image/document-open.png</normaloff>:/image/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load …</string>
   </property>
   <property name="toolTip">
    <string>Load TRS80 / Colour Genie cassette file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="cass80.qrc">
     <normaloff>:/image/application-exit.png</normaloff>:/image/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="cass80.qrc">
     <normaloff>:/image/document-save.png</normaloff>:/image/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save …</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Undo_lmoffset">
   <property name="icon">
    <iconset resource="cass80.qrc">
     <normaloff>:/image/lmoffset-undo.png</normaloff>:/image/lmoffset-undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo &amp;lmoffset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_Information">
   <property name="icon">
    <iconset resource="cass80.qrc">
     <normaloff>:/image/dialog-information.png</normaloff>:/image/dialog-information.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Information …</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="icon">
    <iconset resource="cass80.qrc">
     <normaloff>:/image/preferences.png</normaloff>:/image/preferences.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About …</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="cass80.qrc"/>
 </resources>
 <connections/>
</ui>
